rinja/src/filters/builtin.rs

```diff
@@ -20,6 +20,25 @@ const MAX_LEN: usize = 10_000;
 /// {{ value|fmt("{:?}") }}
 /// ```
 ///
+/// ```
+/// # #[cfg(feature = "code-in-doc")] {
```

Comment:
I don't think that's what @Kijewski meant when they suggested to add doctests. Why not using `source` instead of a feature inside a doctest? Would make tests smaller and less complex.

Reply:
I deferred to the example in the issue ([builtin.rs#L53-L72](https://github.com/rinja-rs/rinja/blob/80bb96931b1a325d0ce74af032d62b222c39a233/rinja/src/filters/builtin.rs#L53-L72)) when adding these. I certainly see why `source` would be a simpler solution though. If that is what you and @Kijewski have in mind, I can update these to use `source` instead of the `code-in-doc` feature. Thanks!

Reply:
It's fine. Let's wait for confirmation so you don't update if it's what they have in mind.

Reply:
Sorry for the late answer. I guess I caught Corona, and the coughing is kinda not so great when you need a good night's sleep to have a working brain. :|

Actually, I meant it this way, in a code block, because IMHO it's much more readable in the generated documentation pages.

If you dislike the parsing overhead a lot, @GuillaumeGomez, then I'm OK with the source attribute, too.

Reply:
If it's what you had mind, then it's fine for me too.

Reply:
Don't forget to rest @Kijewski !